src/main/java/wedlog/address/model/tag/DietaryRequirement.java

```diff
@@ -0,0 +1,48 @@
+package wedlog.address.model.tag;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Guest's dietary requirement in WedLog.
+ * Guarantees: immutable; is always valid
+ */
+public class DietaryRequirement {
+
+    public final String value;
```

Comment:
should we add constraints to the string? (eg alphanumeric) unless you think dietary requirements may need special characters?